enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -43,23 +43,17 @@
 
   Takes an options map which is passed on to [[serdes.base/extract-all]] for each model. The options are documented
   there."
-  [opts]
+  [{:keys [data-model-only include-field-values] :as opts}]
   (log/tracef "Extracting Metabase with options: %s" (pr-str opts))
   (serdes.backfill/backfill-ids)
   ;; TODO document and test data-model-only if we want to keep this feature...
-  (let [model-pred (cond
-                     (:data-model-only opts)
-                     #{"Database" "Dimension" "Field" "FieldValues" "Metric" "Segment" "Table"}
-
-                     (:include-field-values opts)
-                     (constantly true)
-
-                     :else
-                     (complement #{"FieldValues"}))
+  (let [models (cond-> (if data-model-only
+                         (conj serdes.models/data-model "Dimension")
```

Comment:
`Dimension` should not be needed here. It wasn't in the original `exported-models` because they're inlined onto `Field`s, which are part of the new `data-model` set.

Reply:
Sorry, I think I misunderstood. Adding `Dimension` to the `data-model` set breaks things in other places where `data-model` is used along. That's why `Dimension` is here now.